pkg/tfbridge/info.go

```diff
@@ -24,22 +24,23 @@ import (
-	Keywords       []string                   // an optional list of keywords to help discovery of this package.
-	License        string                     // the license, if any, the resulting package has (default is none).
-	Homepage       string                     // the URL to the project homepage.
-	Repository     string                     // the URL to the project source code repository.
-	Config         map[string]*SchemaInfo     // a map of TF name to config schema overrides.
-	ExtraConfig    map[string]*ConfigInfo     // a list of Pulumi-only configuration variables.
-	Resources      map[string]*ResourceInfo   // a map of TF name to Pulumi name; standard mangling occurs if no entry.
-	DataSources    map[string]*DataSourceInfo // a map of TF name to Pulumi resource info.
-	JavaScript     *JavaScriptInfo            // optional overlay information for augmented JavaScript code-generation.
-	Python         *PythonInfo                // optional overlay information for augmented Python code-generation.
-	Golang         *GolangInfo                // optional overlay information for augmented Golang code-generation.
+	P                 *schema.Provider           // the TF provider/schema.
+	Name              string                     // the TF provider name (e.g. terraform-provider-XXXX).
+	ResourcePrefix    string                     // the prefix on resources the provider exposes, if different to `Name`.
+	GitHubOrg         string                     // the GitHub org of the provider. Defaults to `terraform-providers`.
+	Description       string                     // an optional descriptive overview of the package (a default supplied).
+	Keywords          []string                   // an optional list of keywords to help discovery of this package.
+	License           string                     // the license, if any, the resulting package has (default is none).
+	Homepage          string                     // the URL to the project homepage.
+	Repository        string                     // the URL to the project source code repository.
+	Config            map[string]*SchemaInfo     // a map of TF name to config schema overrides.
+	ExtraConfig       map[string]*ConfigInfo     // a list of Pulumi-only configuration variables.
+	Resources         map[string]*ResourceInfo   // a map of TF name to Pulumi name; standard mangling occurs if no entry.
+	DataSources       map[string]*DataSourceInfo // a map of TF name to Pulumi resource info.
+	JavaScript        *JavaScriptInfo            // optional overlay information for augmented JavaScript code-generation.
+	Python            *PythonInfo                // optional overlay information for augmented Python code-generation.
+	Golang            *GolangInfo                // optional overlay information for augmented Golang code-generation.
+	TFProviderVersion string                     // the version of the TF provider on which this was based
```

Comment:
Would you mind adding this to the serializable version of `ProviderInfo` as well? This could be useful for `tf2pulumi`.

Reply:
Added as requested!